src/lib/EventListener/UserMenuListener.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\EventListener;
+
+use EzSystems\EzPlatformAdminUi\Menu\Event\ConfigureMenuEvent;
+use JMS\TranslationBundle\Translation\TranslationContainerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use JMS\TranslationBundle\Model\Message;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
```

Comment:
nitpick: Unordered imports

Reply:
I think we totally do not care about that, as it is not forced by code style.

Reply:
Code Style package version was not updated :disappointed: